stacker/blueprints/__init__.py

```diff
@@ -0,0 +1,33 @@
+from stacker.blueprints.base import Blueprint
+
+from troposphere import Ref
```

Comment:
I'd replace `Ref` with `NoValue` and just use that later, since that's the only use of Ref.